recipes/ecell/meta.yml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: ecell
+  version: "4.1.4"
+build:
+  number: 0
+  skip: True  # [win32]
+  skip: True  # [win64 and py<35]
+  skip: True  # [osx and py34]
+  skip: True  # [py26]
+  skip: True  # [py33]
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - matplotlib
+    - notebook
+    - hdf5 = 1.8.17
```

Comment:
Can you remove this pin, please?
We manage these pinnings in https://github.com/conda-forge/conda-forge-pinning-feedstock